src/main/java/ai/lucidtech/las/sdk/Credentials.java

```diff
@@ -75,7 +76,40 @@ public String getApiEndpoint() {
         return apiEndpoint;
     }
 
-    private Map readFromEnviron() {
+    private void setCredentials(String credentialsPath) {
```

Comment:
This method name + signature might be a bit misleading since it looks at ENV first (It sounds like we are setting the credentials from the path, but that is not always the case, which might be misleading). I suggest renaming it or removing the credentialsPath argument and move :
```java
        String homeDir = System.getProperty("user.home");
        Path credentialsPath = Paths.get(homeDir, ".lucidtech", "credentials.cfg");
```
inside the function